### PR TITLE
Adding PATO software

### PR DESCRIPTION
**Pull request by fsemerar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pato/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp
+    - make
+    - wget
+    - cmake 
+    - boost 
+    - metis
+    - cgal
+    - scotch
```
Comment (jakirkham) [inline recipes/pato/meta.yaml:27]: Dependencies that are linked to should be in `host`. `build` is only for build tools

```suggestion
  host:
    - boost 
    - metis
    - cgal
    - scotch
```

Comment (jakirkham) [reply recipes/pato/meta.yaml:27]: Also some of these may be needed in `run` as well

File: recipes/pato/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  host:
```
Comment (jakirkham) [inline recipes/pato/meta.yaml:29]: To merge with the others

```suggestion
```

File: recipes/pato/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  run:
+    - openmpi
+    # - python
+
+about:
+  home: https://pato.ac
+  license: NASA-1.3
+  summary: 'A package to ...'
+  description: |
+    describe
```
Comment (jakirkham) [inline recipes/pato/meta.yaml:43]: Would be good to fill out at least the `summary`. The `description` is a longer form section, but is optional and can be dropped

File: recipes/pato/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - openmpi
+    # - python
+
+  run:
+    - openmpi
+    # - python
+
+about:
+  home: https://pato.ac
+  license: NASA-1.3
```
Comment (jakirkham) [inline recipes/pato/meta.yaml:40]: Please make sure the `license_file` is listed here as well to package license as well

File: recipes/pato/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - wget
+    - cmake 
+    - boost 
+    - metis
+    - cgal
+    - scotch
+
+  host:
+    - qt
+    - openmpi
```
Comment (jakirkham) [inline recipes/pato/meta.yaml:31]: Please see [the doc section on MPI support]( https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi )

File: recipes/pato/meta.yaml (hunk 1 above)
Comment (fsemerar) [reply recipes/pato/meta.yaml:27]: Thank you so much for your help! I’ll try your recommendations now

File: recipes/pato/meta.yaml (hunk 3 above)
Comment (fsemerar) [reply recipes/pato/meta.yaml:43]: I will complete all these sections, but we wanted to first try if it’s possible to mount a volume (see build.sh commands) on the pipelines, before polishing the rest of the recipe.

File: recipes/pato/build.sh [hunk 6]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e  # exit when any command fails
+
+echo -e "\n### INSTALLING PATO ###\n"
+rm -rf volume
+mkdir volume
+hdiutil attach -mountpoint volume pato_releases_conda.sparsebundle
```
Comment (jakirkham) [inline recipes/pato/build.sh:7]: Is there a reason we are not building from source?

Comment (fsemerar) [reply recipes/pato/build.sh:7]: PATO uses a modified version of an older release of OpenFoam, which needs to be patched in multiple ways. I will double check with the main developer of PATO (@jmeurisse) if building from source could be a viable option.

File: recipes/pato/meta.yaml (hunk 1 above)
Comment (fsemerar) [reply recipes/pato/meta.yaml:29]: After making this change, the osx pipeline is still failing in the same spot

**Comment by fsemerar** [comment]
Hi Jeremie, do you accept to be one of the maintainers of this recipe?  @jmeurisse

File: recipes/pato/build.sh (hunk 6 above)
Comment (jmeurisse) [reply recipes/pato/build.sh:7]: We are building from source. The Mac OS installation needs to happen in a disk image.

**Comment by jmeurisse** [comment]
> Hi Jeremie, do you accept to be one of the maintainers of this recipe? @jmeurisse

Yes

**Pull request closed without merging**